Moggles/BackgroundServices/ScheduledFeatureTogglesService.cs

```diff
@@ -72,6 +80,17 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                                         }
                                     }
                                     await _appRepository.UpdateAsync(app);
+                                    if (toggleSchedule.ForceCacheRefresh == true)
```

Comment:
You don't necessarily need the '== true'. Just checking  if (toggleSchedule.ForceCacheRefresh) { ... } will also work & it is simpler & clear